foundry.toml

```diff
@@ -3,4 +3,7 @@ src = 'src'
 out = 'out'
 libs = ['lib']
 
+[profile.hf]
```

Comment:
New .yml has to be added under .github folder for this to be actively running. There is a example in vaults repo